setup.py

```diff
@@ -36,7 +36,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='ora2',
-    version='3.6.11',
+    version='3.6.12',
```

Comment:
It looks like the release still exists on pypi. We might have to bump this again to 3.6.13